gcov builds are failing to configure

```console

Compiler Setup...

-- Looking for OpenMP...
-- Looking for OpenMP... -fopenmp
-- Performing Test HAS_MARCH_NATIVE
-- Performing Test HAS_MARCH_NATIVE - Success
-- Performing Test HAS_WNOEXCEPT
-- Performing Test HAS_WNOEXCEPT - Success
-- Performing Test HAS_WSUGGEST_ATTRIBUTE
-- Performing Test HAS_WSUGGEST_ATTRIBUTE - Success
-- Performing Test HAS_WUNUSED_LOCAL_TYPEDEFS
-- Performing Test HAS_WUNUSED_LOCAL_TYPEDEFS - Success
CMake Error at config/compilerEnv.cmake:653 (message):
  When calling

  toggle_compiler_flag(switch, compiler_flag, compiler_flag_var_names),

  compiler_flag_var_names must be set to one or more of these valid

  names: C;CXX;EXE_LINKER.
Call Stack (most recent call first):
  config/compilerEnv.cmake:261 (toggle_compiler_flag)
  CMakeLists.txt:87 (dbsSetupCxx)


-- Configuring incomplete, errors occurred!
See also "/scratch/regress/cdash/draco/Nightly_gcc-gcov-develop/Coverage/build/CMakeFiles/CMakeOutput.log".
See also "/scratch/regress/cdash/draco/Nightly_gcc-gcov-develop/Coverage/build/CMakeFiles/CMakeError.log".
```

--- a/config/compilerEnv.cmake
+++ b/config/compilerEnv.cmake
@@ -649,11 +649,12 @@ macro( toggle_compiler_flag switch compiler_flag
     if( NOT ${comp} STREQUAL "C" AND
         NOT ${comp} STREQUAL "CXX" AND
         NOT ${comp} STREQUAL "Fortran" AND
-        NOT ${comp} STREQUAL "EXE_LINKER")
-      message(FATAL_ERROR "When calling
-toggle_compiler_flag(switch, compiler_flag, compiler_flag_var_names),
-compiler_flag_var_names must be set to one or more of these valid
-names: C;CXX;EXE_LINKER.")
+        NOT ${comp} STREQUAL "EXE_LINKER" AND
+        NOT ${comp} STREQUAL "SHARED_LINKER")
+      message(FATAL_ERROR "When calling "
+"toggle_compiler_flag(switch, compiler_flag, compiler_flag_var_names), "
+"compiler_flag_var_names must be set to one or more of these valid "
+"names: C;CXX;EXE_LINKER.")
     endif()
 
     string( REPLACE "+" "x" safe_CMAKE_${comp}_FLAGS

